main.py

```diff
@@ -122,15 +122,38 @@ def calc_grid_cant(self):
     def set_point_color(self, color):
         self.point_color = color
         self.draw_grid()
+        self.draw_board()
 
     def set_back_color(self, color):
         self.back_color = color
         self.draw_grid()
+        self.draw_board()
 
     def set_line_color(self, color):
         self.line_color = color
         self.draw_grid()
-
+        self.draw_board()
+
+    def draw_board(self):
+        for box_row in self.boxes:
+            for box in box_row:
+                box_tr = (box.pos_x + self.box_size[0]/2, box.pos_y - self.box_size[1]/2)
+                box_tl = (box.pos_x - self.box_size[0]/2, box.pos_y - self.box_size[1]/2)
+                box_br = (box.pos_x + self.box_size[0]/2, box.pos_y + self.box_size[1]/2)
+                box_bl = (box.pos_x - self.box_size[0]/2, box.pos_y + self.box_size[1]/2)
```

Comment:
Could you use floor division when dividing by ints?

Reply:
yes, I have changed these to floor division as we're playing pixel coordinates